poc/tests/test_vdaf.py

```diff
@@ -122,6 +124,61 @@ def test_fields(self):
             self.assertEqual(pine.flp_norm_equality.field, FieldType)
             self.assertEqual(pine.flp.field, FieldType)
 
+    def test_params_combo(self):
+        class PineTest(Pine[Field64, XofTurboShake128]):
```

Comment:
nit: I don't think it's necessary to define a sub-class. For this test you could just call `Pine()` I think.

Reply:
Good call. I have to remove the `@abstractmethod` decoration on `__init__` for this to work.